config/initializers/assets.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile += ['payments.js']
```

Comment:
The name of the js file isn't `payments`, so this won't work - it should be `payment.js` - look it up.